File: NextApp/Android/core/testing/src/main/kotlin/jp/komehyappyo/member/next/core/testing/SampleData.kt
package jp.komehyappyo.member.next.core.testing

import jp.komehyappyo.member.next.core.model.Schedule
import java.time.Instant

object SampleData {
    val schedule = Schedule(
        userId = "guest",
        title = "サンプル予定",
        startDateTime = Instant.parse("2026-07-23T01:00:00Z"),
        endDateTime = Instant.parse("2026-07-23T02:00:00Z"),
    )
}

File: NextApp/iOS/OrgPortalNext/Core/Testing/SampleData.swift
import Foundation
import Model

public enum SampleData {
    public static func schedule(
        title: String = "サンプル予定",
        start: Date = .now
    ) -> Schedule {
        Schedule(
            userId: "guest_example",
            title: title,
            startDateTime: start,
            endDateTime: start.addingTimeInterval(3600),
            timeOfDay: .specified
        )
    }
}
